src/entries/popup/pages/home/TokenDetails/About.tsx

```diff
@@ -229,6 +233,49 @@ function FullyDilutedInfoRow({ fullyDiluted }: { fullyDiluted: ReactNode }) {
   );
 }
 
+const LinkInline = ({
```

Comment:
`TextLink` component wasn't enough?

Reply:
Are we using components created in the design-system docs elsewhere? I thought these were for display purposes only, like when they bring out a cart of desserts for you to look at.